cmd/di.go

```diff
@@ -90,12 +90,27 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 	di.bootstrapLocationComponents(nodeOptions.Location, nodeOptions.Directories.Config)
 	di.bootstrapNodeComponents(nodeOptions)
 
+	if err := di.Node.Start(); err != nil {
+		return err
+	}
+
 	return nil
 }
 
 // Shutdown stops container
-func (di *Dependencies) Shutdown() {
+func (di *Dependencies) Shutdown() error {
+	if di.ServiceManager != nil {
+		if err := di.ServiceManager.Kill(); err != nil {
```

Comment:
If `ServiceManager.Kill()`  returns error, node is left unstopped

Reply:
brefore:
```
func() error {	
		errorServiceManager := di.ServiceManager.Kill()	
		errorNode := di.Node.Kill()	
		di.Shutdown()	
 		if errorServiceManager != nil {	
			return errorServiceManager	
		}	
		return errorNode	
	}
```

Reply:
Changed it. 
Collecting all errors from all `Kill` commands, logging all of them at the end, and returning only the first error.